doc/BACKUP.md

```diff
@@ -0,0 +1,413 @@
+
+Recovery of the `hsm_secret` is sufficient to recover any onchain
+funds.
+Recovery of the `hsm_secret` is necessary, but insufficient, to recover
+any in-channel funds.
+To recover in-channel funds, you need to use one or more of the other
+backup strategies below.
+
+Database File Backups
+---------------------
+
+This is the least desirable backup strategy, as it *can* lead to loss
+of all in-channel funds if you use it.
+However, having *no* backup strategy at all *will* lead to loss of all
+in-channel funds, so this is still better than nothing.
+
+While `lightningd` is not running, just copy the `lightningd.sqlite3` file
+in the `$LIGHTNINGDIR` on backup media somewhere.
+
+To recover, just copy the backed up `lightningd.sqlite3` into your new
+`$LIGHTNINGDIR` together with the `hsm_secret`.
+
+You can also use any automated backup system as long as it includes the
+`lightningd.sqlite3` file (and optionally `hsm_secret`, but note that
+as a secret key, thieves getting a copy of your backups may allow them
+to steal your funds, even in-channel funds).
+
+This backup method is undesirable, since it cannot recover the following
+channels:
+
+* Channels with peers that do not support `option_dataloss_protect`.
+  * Most nodes on the network already support `option_dataloss_protect`
+    as of November 2020.
+  * If the peer does not support this option, then the entire channel
+    funds will be revoked by the peer.
+  * Specially-coded peers can pretend to support this option, but end
+    up taking the entire channel funds (or otherwise more than what it
+    should have) when you try to recover.
+    Standard releases of Lightning Network node software do not do this
+    (they just support this option according to the standard) but note
+    that anyone can modify their node software, especially with our
+    well-documented open-source code.
+* Channels created *after* the copy was made are not recoverable.
+  * Data for those channels does not exist in the backup, so your node
+    cannot recover them.
+
+Because of the above, this strategy is discouraged: you *can* potentially
+lose all funds in open channels.
+
+However, again, note that a "no backups #reckless" strategy leads to
+*definite* loss of funds, so you should still prefer *this* strategy rather
+than having *no* backups at all.
+
+Even if you have the better options below, you might still want to do
+this as a worst-case fallback, as long as you:
+
+* Attempt to recover using the other backup options below first.
+  Any one of them will be better than this backup option.
+* Recover by this method **ONLY** as a ***last*** resort.
+* Recover using the most recent backup you can find.
+  Take time to look for the most recent available backup.
+
+Again, this strategy can lead to only ***partial*** recovery of funds,
+or even to complete failure to recover, so use the other methods below
+first to recover!
+
+### Backing Up While `lightningd` Is Running
```

Comment:
Ok, while the above should be the last option presented, this one should be even laster! :wink: